realm/src/androidTest/java/io/realm/NotificationsTest.java

```diff
@@ -483,4 +483,43 @@ public void onChange() {
         assertEquals(0, counter.get());
         assertEquals(0, realm.getChangeListeners().size());
     }
+
+    // different instances of Realm sharing the same configuration
+    // should have different handlers
+    public void testSameConfigurationDifferentHandler() throws InterruptedException {
```

Comment:
This test has been superseeded by https://github.com/realm/realm-java/pull/1469 and should be removed.